src/preferences/broadcastsettings_legacy.cpp

```diff
@@ -1,5 +1,7 @@
 #include "preferences/broadcastsettings.h"
+#include "broadcast/defs_broadcast.h"
 
+#define BROADCAST_FORMAT_OV_LEGACYNAME "OggVorbis"
```

Comment:
We should not define this constant twice.

In this case it should be moved into the header file with a comment why it is needed and that it shall only be used for this special purpose.